### PR TITLE
MON-18599 - Documentation - Add Azure Sqldatabase cpu and memory mode

### PR DESCRIPTION
**Pull request by lucie-dubrunfaut**
## Description

Updating Azure Sqldatabase documentation

## Target version (i.e. version that this PR changes)

- [ ] 21.10.x (staging)
- [ ] 22.04.x (staging)
- [ ] 22.10.x (staging)
- [ ] 23.04.x (staging)
- [ ] Cloud (staging)
- [x] Monitoring Connectors (staging)
- [ ] 23.10.x (next)

### CODE REVIEW

**Comment by lucie-dubrunfaut** [comment]
WARNING: macro '--server' is not defined in host template(s)
WARNING: macro '--subscription' is not defined in host template(s)
WARNING: macro '--tenant' is not defined in host template(s)
WARNING: macro '--client-id' is not defined in host template(s)
WARNING: macro '--client-secret' is not defined in host template(s)
CRITICAL: option '' in Monitoring Connector commands not available for this plugin

By using doc generator script.

**Review by omercier — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md [hunk 1]
```diff
@@ -94,138 +124,578 @@ pour en savoir plus sur la découverte automatique d'hôtes.
...
+| INTERVAL          |                                                                                                     | PT5M              |             |
+| AGGREGATION       |                                                                                                     | Total             |             |
+| WARNINGAPPCPU     | Warning threshold where '*'                                                                         |                   |             |
+| CRITICALAPPCPU    | Critical threshold where '*'                                                                        |                   |             |
+| WARNINGAPPMEMORY  | Warning threshold where '*'                                                                         |                   |             |
+| CRITICALAPPMEMORY | Critical threshold where '*'                                                                        |                   |             |
+| EXTRAOPTIONS      | Any extra option you may want to add to the command (E.g. a --verbose flag). Tous les options sont listées [ici](#options-disponibles) |                   |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md:288]:
```suggestion
| EXTRAOPTIONS      | Any extra option you may want to add to the command (E.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles) |                   |             |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md [hunk 2]
```diff
@@ -94,138 +124,578 @@ pour en savoir plus sur la découverte automatique d'hôtes.
...
+| TIMEFRAME            |                                                                                                     | 900               |             |
+| INTERVAL             |                                                                                                     | PT5M              |             |
+| AGGREGATION          |                                                                                                     | Total             |             |
+| FILTERMETRIC         |                                                                                                     |                   |             |
+| WARNINGUSAGEPERCENT  | Thresholds where '*'                                                                                |                   |             |
+| CRITICALUSAGEPERCENT | Thresholds where '*'                                                                                |                   |             |
+| EXTRAOPTIONS         | Any extra option you may want to add to the command (E.g. a --verbose flag). Tous les options sont listées [ici](#options-disponibles) |                   |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md:317]:
```suggestion
| EXTRAOPTIONS         | Any extra option you may want to add to the command (E.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles) |                   |             |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md [hunk 3]
```diff
@@ -94,138 +124,578 @@ pour en savoir plus sur la découverte automatique d'hôtes.
...
+|:------------------|:----------------------------------------------------------------------------------------------------|:------------------|:------------|
+| TIMEFRAME         |                                                                                                     | 900               |             |
+| INTERVAL          |                                                                                                     | PT5M              |             |
+| AGGREGATION       |                                                                                                     | Total             |             |
+| WARNINGDEADLOCKS  | Warning threshold                                                                                   |                   |             |
+| CRITICALDEADLOCKS | Critical threshold                                                                                  |                   |             |
+| EXTRAOPTIONS      | Any extra option you may want to add to the command (E.g. a --verbose flag). Tous les options sont listées [ici](#options-disponibles) |                   |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md:329]:
```suggestion
| EXTRAOPTIONS      | Any extra option you may want to add to the command (E.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles) |                   |             |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md [hunk 4]
```diff
@@ -94,138 +124,578 @@ pour en savoir plus sur la découverte automatique d'hôtes.
...
+| AGGREGATION    |                                                                                                                                          | Total                        |             |
+| STATUSUNKNOWN  | Set unknown threshold for status (Default: '%{status} =~ /^Unknown$/').  You can use the following variables: %{status}, %{summary}      |                              |             |
+| STATUSCRITICAL | Set critical threshold for status (Default: '%{status} =~ /^Unavailable$/').  You can use the following variables: %{status}, %{summary} | %{status} =~ /^Unavailable$/ |             |
+| STATUSWARNING  | Set warning threshold for status (Default: '').  You can use the following variables: %{status}, %{summary}                              |                              |             |
+| EXTRAOPTIONS   | Any extra option you may want to add to the command (E.g. a --verbose flag). Tous les options sont listées [ici](#options-disponibles)                                      |                              |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md:343]:
```suggestion
| EXTRAOPTIONS   | Any extra option you may want to add to the command (E.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles)                                      |                              |             |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md [hunk 5]
```diff
@@ -94,138 +124,578 @@ pour en savoir plus sur la découverte automatique d'hôtes.
...
+| STATUSOK             |                                                                                                     | %{status} =~ /^Available$/   |             |
+| FILTERMETRIC         |                                                                                                     |                              |             |
+| STATUSCRITICAL       |                                                                                                     | %{status} =~ /^Unavailable$/ |             |
+| WARNINGUSAGEPERCENT  | Thresholds where '*'                                                                                |                              |             |
+| CRITICALUSAGEPERCENT | Thresholds where '*'                                                                                |                              |             |
+| EXTRAOPTIONS         | Any extra option you may want to add to the command (E.g. a --verbose flag). Tous les options sont listées [ici](#options-disponibles) |                              |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md:358]:
```suggestion
| EXTRAOPTIONS         | Any extra option you may want to add to the command (E.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles) |                              |             |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md [hunk 6]
```diff
@@ -94,138 +124,578 @@ pour en savoir plus sur la découverte automatique d'hôtes.
...
+|:-----------------|:----------------------------------------------------------------------------------------------------|:------------------|:------------|
+| TIMEFRAME        |                                                                                                     | 900               |             |
+| INTERVAL         |                                                                                                     | PT5M              |             |
+| AGGREGATION      |                                                                                                     | Average           |             |
+| WARNINGSESSIONS  | Warning threshold                                                                                   |                   |             |
+| CRITICALSESSIONS | Critical threshold                                                                                  |                   |             |
+| EXTRAOPTIONS     | Any extra option you may want to add to the command (E.g. a --verbose flag). Tous les options sont listées [ici](#options-disponibles) |                   |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md:370]:
```suggestion
| EXTRAOPTIONS     | Any extra option you may want to add to the command (E.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles) |                   |             |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md [hunk 7]
```diff
@@ -94,138 +124,578 @@ pour en savoir plus sur la découverte automatique d'hôtes.
...
+| INTERVAL             |                                                                                                     | PT5M              |             |
+| AGGREGATION          |                                                                                                     | Maximum           |             |
+| WARNINGUSAGEBYTES    | Warning threshold where '*'                                                                         |                   |             |
+| CRITICALUSAGEBYTES   | Critical threshold where '*'                                                                        |                   |             |
+| WARNINGUSAGEPERCENT  | Warning threshold where '*'                                                                         |                   |             |
+| CRITICALUSAGEPERCENT | Critical threshold where '*'                                                                        |                   |             |
+| EXTRAOPTIONS         | Any extra option you may want to add to the command (E.g. a --verbose flag). Tous les options sont listées [ici](#options-disponibles) |                   |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md:384]:
```suggestion
| EXTRAOPTIONS         | Any extra option you may want to add to the command (E.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles) |                   |             |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md [hunk 8]
```diff
@@ -94,138 +124,578 @@ pour en savoir plus sur la découverte automatique d'hôtes.
...
+|:----------------|:----------------------------------------------------------------------------------------------------|:------------------|:------------|
+| TIMEFRAME       |                                                                                                     | 900               |             |
+| INTERVAL        |                                                                                                     | PT5M              |             |
+| AGGREGATION     |                                                                                                     | Average           |             |
+| WARNINGWORKERS  | Warning threshold                                                                                   |                   |             |
+| CRITICALWORKERS | Critical threshold                                                                                  |                   |             |
+| EXTRAOPTIONS    | Any extra option you may want to add to the command (E.g. a --verbose flag). Tous les options sont listées [ici](#options-disponibles) |                   |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md:396]:
```suggestion
| EXTRAOPTIONS    | Any extra option you may want to add to the command (E.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles) |                   |             |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md [hunk 9]
```diff
@@ -94,138 +124,578 @@ pour en savoir plus sur la découverte automatique d'hôtes.
...
+| --proxyurl         | Proxy URL if any                                                                                                                       | Azcli       |
+| --filter-dimension | Specify the metric dimension (required for some specific metrics) Syntax example: --filter-dimension="$metricname eq '$metricvalue'"   | Custom mode |
+| --per-sec          | Display the statistics based on a per-second period.                                                                                   | Custom mode |
+
+</TabItem>
+</Tabs>
+
+#### Options des modes
+
+Les options spécifiques aux modes sont listées ci-dessus :
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md:590]:
```suggestion
Les options spécifiques aux modes sont listées ci-dessous :
```

File: pp/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md [hunk 10]
```diff
@@ -5,224 +5,695 @@ title: Azure SQL Database
...
-
-## Pack Assets
+## Pack assets
 
 ### Templates
 
-The Centreon Monitoring Connector **Azure SQL Database** brings a host template:
+The Monitoring Connector **Azure SQL Database** brings a host template:
+
+* **Cloud-Azure-Database-SqlDatabase**
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md:14]:
```suggestion
* **Cloud-Azure-Database-SqlDatabase-custom**
```

File: pp/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md [hunk 11]
```diff
@@ -5,224 +5,695 @@ title: Azure SQL Database
...
+
+The connector brings the following service templates (sorted by the host template they are attached to):
 
-* Cloud-Azure-Database-SqlDatabase-custom
+<Tabs groupId="sync">
+<TabItem value="Cloud-Azure-Database-SqlDatabase" label="Cloud-Azure-Database-SqlDatabase">
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md:19]:
```suggestion
<TabItem value="Cloud-Azure-Database-SqlDatabase-custom" label="Cloud-Azure-Database-SqlDatabase-custom">
```

File: pp/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md [hunk 12]
```diff
@@ -5,224 +5,695 @@ title: Azure SQL Database
...
+
+| Service Alias | Service Template                        | Service Description                               |
+|:--------------|:----------------------------------------|:--------------------------------------------------|
+| Cpu           | Cloud-Azure-Database-SqlDatabase-Cpu    | Check Azure Database for Sqldatabase CPU usage    |
+| Health        | Cloud-Azure-Database-SqlDatabase-Health | Check database state                              |
+| Memory        | Cloud-Azure-Database-SqlDatabase-Memory | Check Azure Database for SqlDatabase memory usage |
 
-It brings the following service templates:
+> The services listed above are created automatically when the **Cloud-Azure-Database-SqlDatabase** host template is used.
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md:27]:
```suggestion
> The services listed above are created automatically when the **Cloud-Azure-Database-SqlDatabase-custom** host template is used.
```

File: pp/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md [hunk 13]
```diff
@@ -5,224 +5,695 @@ title: Azure SQL Database
...
+| Service Alias | Service Template                               | Service Description               |
+|:--------------|:-----------------------------------------------|:----------------------------------|
+| App-Resources | Cloud-Azure-Database-SqlDatabase-App-Resources | Check App CPU and memory metrics  |
+| Connections   | Cloud-Azure-Database-SqlDatabase-Connections   | Check the number of connections   |
+| Deadlocks     | Cloud-Azure-Database-SqlDatabase-Deadlocks     | Check database for deadlocks      |
+| Sessions      | Cloud-Azure-Database-SqlDatabase-Sessions      | Check sessions usage              |
+| Storage       | Cloud-Azure-Database-SqlDatabase-Storage       | Check database size               |
+| Workers       | Cloud-Azure-Database-SqlDatabase-Workers       | Check the number of workers       |
+
+> The services listed above are not created automatically when a host template is applied. To use them, [create a service manually](/docs/monitoring/basic-objects/services), then appy the service template you want.
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md:41]:
```suggestion
> The services listed above are not created automatically when a host template is applied. To use them, [create a service manually](/docs/monitoring/basic-objects/services), then apply the service template you want.
```

File: pp/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md [hunk 14]
```diff
@@ -5,224 +5,695 @@ title: Azure SQL Database
...
-> Two methods can be used to set the macros:
+### Using a host template provided by the connector
 
->> * Full ID of the Resource (`/subscriptions/<subscription_id>/resourceGroups/<resourcegroup_id>/providers/XXXXX/XXXXX/<resource_name>`)
-in **AZURERESOURCE**
-> * Resource name in the **AZURERESOURCE** macro, and resource group name in the **AZURERESOURCEGROUP** macro.
+1. Log into Centreon and add a new host through **Configuration > Hosts**.
+2. In the **IP Address/DNS** field, set the following IP address: **127.0.0.1**.
+3. Apply the **Cloud-Azure-Database-SqlDatabase-custom** template to the host. A list of macros appears. Macros allow you to define how the connector will connect to the resource, and to customize the connector's behavior.
+4. Fill in the macros you want. Some macros are mandatory. For example, for this connector, you must define the **AZURECUSTOMMODE** macros (possible values are **api** or **azcli**). Indeed, 2 modes of communication cn be used with this resource: either using the command tool azcli, or by querying the API directly.
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md:230]:
```suggestion
4. Fill in the macros you want. Some macros are mandatory. For example, for this connector, you must define the **AZURECUSTOMMODE** macros (possible values are **api** or **azcli**). Indeed, 2 modes of communication can be used with this resource: either using the command tool azcli, or by querying the API directly.
```

File: pp/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md [hunk 15]
```diff
@@ -5,224 +5,695 @@ title: Azure SQL Database
...
 
-| Mandatory   | Macro              | Description                                   |
-|:------------|:-------------------|:----------------------------------------------|
-|             | AZURECLICUSTOMMODE | Custom mode **azcli**                         |
-|             | AZURERESOURCE      | ID or name of the Azure SQL Database resource |
-|             | AZURERESOURCEGROUP | Resource group name if resource name is used  |
-|             | AZURESUBSCRIPTION  | Subscription ID                               |
+| Macro              | Description                                                                                                   | Default value     | Mandatory   |
+|:-------------------|:--------------------------------------------------------------------------------------------------------------|:------------------|:------------|
+| AZURECUSTOMMODE    | When a plugin offers several ways  to get the an information the desired one must be defined with this option | api               |             |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md:253]:
```suggestion
| AZURECUSTOMMODE    | When a plugin offers several ways to get information the desired one must be defined with this option | api               |             |
```

File: pp/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md [hunk 16]
```diff
@@ -5,224 +5,695 @@ title: Azure SQL Database
...
+| AZURERESOURCE      | Set resource name or id . It is the database name                                                             |                   |             |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md:254]:
```suggestion
| AZURERESOURCE      | Set resource name or ID. It is the database name                                                             |                   |             |
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md [hunk 17]
```diff
@@ -5,43 +5,62 @@ title: Azure SQL Database
...
-| Sessions      | Cloud-Azure-Database-SqlDatabase-Sessions      | Contrôle le nombre de sessions utilisées        |        |
-| Storage       | Cloud-Azure-Database-SqlDatabase-Storage       | Contrôle la taille de la base                   |        |
-| Workers       | Cloud-Azure-Database-SqlDatabase-Workers       | Contrôle le nombre de workers                   |        |
+</TabItem>
+<TabItem value="Non rattachés à un modèle d'hôte" label="Non rattachés à un modèle d'hôte">
+
+| Alias         | Modèle de service                              | Description                                     |
+|:--------------|:------------------------------------------------------|:------------------------------------------------|
+| App-Resources | Cloud-Azure-Database-SqlDatabase-App-Resources-custom | Contrôle les métriques CPU et Mémoire de l'App  |
+| Connections   | Cloud-Azure-Database-SqlDatabase-Connections-custom   | Contrôle le nombre de connexion par statut      |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md:36]:
```suggestion
| Connections   | Cloud-Azure-Database-SqlDatabase-Connections-custom   | Contrôle le nombre de connexions par statut      |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md [hunk 18]
```diff
@@ -94,138 +128,560 @@ pour en savoir plus sur la découverte automatique d'hôtes.
...
+
+<Tabs groupId="sync">
+<TabItem value="Azure Monitor API" label="Azure Monitor API">
+
+| Macro              | Description                                                                                                   | Valeur par défaut | Obligatoire |
+|:-------------------|:--------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| AZURECLIENTID      | Set Azure client ID                                                                                           |                   | X           |
+| AZURECLIENTSECRET  | Set Azure client secret                                                                                       |                   | X           |
+| AZURECUSTOMMODE    | When a plugin offers several ways  to get the an information the desired one must be defined with this option | api               |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md:243]:
```suggestion
| AZURECUSTOMMODE    | When a plugin offers several ways to get information, the desired one must be defined with this option | api               |             |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md [hunk 19]
```diff
@@ -94,138 +128,560 @@ pour en savoir plus sur la découverte automatique d'hôtes.
...
+| AZURERESOURCE      | Set resource name or id . It is the database name                                                             |                   |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md:244]:
```suggestion
| AZURERESOURCE      | Set resource name or ID . It is the database name                                                             |                   |             |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md [hunk 20]
```diff
@@ -94,138 +128,560 @@ pour en savoir plus sur la découverte automatique d'hôtes.
...
+</TabItem>
+<TabItem value="Azure AZ CLI" label="Azure AZ CLI">
 
-* Ajoutez un hôte à Centreon depuis la page **Configuration > Hôtes**.
-* Remplissez le champ **Adresse IP/DNS** avec l'adresse **127.0.0.1**.
-* Appliquez le modèle d'hôte **Cloud-Azure-Database-SqlDatabase-custom**.
-* Une fois le modèle appliqué, renseignez les macros correspondantes. Attention, certaines macros sont obligatoires. Elles doivent être renseignées selon le *custom mode* utilisé.
+| Macro              | Description                                                                                                   | Valeur par défaut | Obligatoire |
+|:-------------------|:--------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| AZURECUSTOMMODE    | When a plugin offers several ways  to get the an information the desired one must be defined with this option | api               |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md:257]:
```suggestion
| AZURECUSTOMMODE    | When a plugin offers several ways to get information, the desired one must be defined with this option | api               |             |
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: pp/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md [hunk 21]
```diff
@@ -5,224 +5,681 @@ title: Azure SQL Database
...
-    --proxyurl=''  \
-    --warning-deadlocks='' \
-    --critical-deadlocks='' \
+	--plugin=cloud::azure::database::sqldatabase::plugin \
+	--mode=app-resources \
+	--custommode='api' \
+	--resource='' \
+	--resource-group='' \
+	--subscription='' \
+	--tenant='' \
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md:398]:
```suggestion
	--tenant='xxxxxxxxx' \
```

File: pp/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md [hunk 22]
```diff
@@ -5,224 +5,681 @@ title: Azure SQL Database
...
+	--client-id='' \
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md:399]:
```suggestion
	--client-id='xxxxxxxxx' \
```

File: pp/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md [hunk 23]
```diff
@@ -5,224 +5,681 @@ title: Azure SQL Database
...
+	--client-secret='' \
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md:400]:
```suggestion
	--client-secret='xxxxxxxxx' \
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md [hunk 24]
```diff
@@ -94,138 +128,560 @@ pour en savoir plus sur la découverte automatique d'hôtes.
...
-    --proxyurl=''  \
-    --warning-deadlocks='' \
-    --critical-deadlocks='' \
-    --use-new-perfdata
+	--plugin=cloud::azure::database::sqldatabase::plugin \
+	--mode=app-resources \
+	--custommode='api' \
+	--resource='' \
+	--resource-group='' \
+	--subscription='' \
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md:399]:
```suggestion
	--subscription='xxxxxxxxx' \
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md [hunk 25]
```diff
@@ -94,138 +128,560 @@ pour en savoir plus sur la découverte automatique d'hôtes.
...
+	--tenant='' \
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md:400]:
```suggestion
	--tenant='xxxxxxxxx' \
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md [hunk 26]
```diff
@@ -94,138 +128,560 @@ pour en savoir plus sur la découverte automatique d'hôtes.
...
+	--client-id='' \
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md:401]:
```suggestion
	--client-id='xxxxxxxxx' \
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md [hunk 27]
```diff
@@ -94,138 +128,560 @@ pour en savoir plus sur la découverte automatique d'hôtes.
...
+	--client-secret='' \
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md:402]:
```suggestion
	--client-secret='xxxxxxxxx' \
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md [hunk 28]
```diff
@@ -94,138 +128,560 @@ pour en savoir plus sur la découverte automatique d'hôtes.
...
-    --client-id='xxxxxxxxx' \
-    --client-secret='xxxxxxxxx' \
-    --proxyurl=''  \
-    --warning-deadlocks='' \
-    --critical-deadlocks='' \
-    --use-new-perfdata
+	--plugin=cloud::azure::database::sqldatabase::plugin \
+	--mode=app-resources \
+	--custommode='api' \
+	--resource='' \
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md:397]:
```suggestion
	--resource='SQLDB001A' \
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md [hunk 29]
```diff
@@ -94,138 +128,560 @@ pour en savoir plus sur la découverte automatique d'hôtes.
...
+	--resource-group='' \
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md:398]:
```suggestion
	--resource-group='RSG1234' \
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by omercier — APPROVED** [review]

**Review by cg-tw — APPROVED** [review]

**Pull request merged**